internal/compiler/builtins.slint

```diff
@@ -198,7 +223,9 @@ component WindowItem {
     in property <image> icon;
 }
 
-export component Window inherits WindowItem { }
+export component Window inherits WindowItem {
+    MenuBar {}
```

Comment:
Does this need some layout handling?

Reply:
(How) Does the menu item orientation (row/column) depend on the surrounding layout? I also think making this a little more customizable would make it a little outstanding from the de-facto standard (implicitly anchored to window top). Minimal example:

```slint
export component Window {
    HorizontalLayout {
        MenuBar { /* menu items would now be layed out below each other (like a list/column)  */ }
    }
}
```